### PR TITLE
fix: updated status tag components to chip in page header itself and …

### PR DESCRIPTION
**Pull request by za2201**
## Summary of the changes

Updated the header component tsx to display chip instead of status tag
Updated page header component page codes in guidance and code mdx files to display chip instead of status tag

## Related issue

#316 

## Checklist

- [y] I have [manually accessibility tested](https://design.sis.gov.uk/accessibility/testing/manual-testing) any changes, if relevant.
- [y] I have raised this pull request against the [develop branch](https://github.com/mi6/ic-design-system/tree/develop)

### CODE REVIEW

File: src/content/structured/components/page-header/code.mdx [hunk 1]
```diff
@@ -63,7 +64,7 @@ export const withBreadcrumbNavigation = [
   {
     language: "Web component",
     snippet: `<ic-page-header heading="Coffee recipes" subheading="This is a page header component that has breadcrumb navigation.">
-  <ic-status-tag slot="heading-adornment" label="Beta"></ic-status-tag>
+  <ic-chip slot="heading-adornment" label="Beta"></ic-chip>
```
Comment (ad9242) [inline src/content/structured/components/page-header/code.mdx:67]:
```suggestion
  <ic-chip slot="heading-adornment" label="BETA", size="large"></ic-chip>
```
Wherever there is a chip , can we make it use the label "BETA" (uppercase) and use the "large" sized chip

**Review by ad9242 — CHANGES_REQUESTED** [review]
comment about using size="large" for chips and label="BETA"

File: src/content/structured/components/page-header/code.mdx [hunk 2]
```diff
@@ -130,7 +130,7 @@ export const withActionsInputStepper = [
   {
     language: "React",
     snippet: `<IcPageHeader heading="Coffee recipes" subheading="This is a page header component that has an input area, actions and stepper." reverseOrder>
-  <IcStatusTag slot="heading-adornment" label="Beta" />
+  <IcChip slot="heading-adornment" label="BETA", size="large" />
```
Comment (jd239) [inline src/content/structured/components/page-header/code.mdx:133]:
```suggestion
  <IcChip slot="heading-adornment" label="BETA" size="large" />
```

File: src/content/structured/components/page-header/code.mdx [hunk 3]
```diff
@@ -192,7 +192,7 @@ export const withActionsInputTabs = [
   subheading="This is a page header component that has an input area, actions &amp; tabs." 
   reverse-order="true"
 >
-  <ic-status-tag slot="heading-adornment" label="Beta"></ic-status-tag>
+  <ic-chip slot="heading-adornment" label="BETA", size="large"></ic-chip>
```
Comment (jd239) [inline src/content/structured/components/page-header/code.mdx:195]:
```suggestion
  <ic-chip slot="heading-adornment" label="BETA" size="large"></ic-chip>
```

File: src/content/structured/components/page-header/code.mdx [hunk 4]
```diff
@@ -218,7 +218,7 @@ export const withActionsInputTabs = [
   heading="Coffee recipes" 
   subheading="This is a page header component that has an input area, actions and tabs." 
   reverseOrder>
-  <IcStatusTag slot="heading-adornment" label="Beta" />
+  <IcChip slot="heading-adornment" label="BETA", size="large" />
```
Comment (jd239) [inline src/content/structured/components/page-header/code.mdx:221]:
```suggestion
  <IcChip slot="heading-adornment" label="BETA" size="large" />
```

**Review by jd239 — CHANGES_REQUESTED** [review]
Thanks for working on this ticket. Looks like a rogue `,` is mixed within the code snippets. Once that has been removed, happy to approve.

File: src/content/structured/components/page-header/code.mdx (hunk 2 above)
Comment (ad9242) [reply src/content/structured/components/page-header/code.mdx:133]: oops, sorry - that was me

**Comment by GCHQ-Developer-112** [comment]
Hey @za2201 , please can rebase your branch off develop

**Comment by za2201** [comment]
Hi, rebase has been done

**Review by MI6-451 — DISMISSED** [review]

**Comment by ad9242** [comment]
I think there are just 2 rogue `"label="BETA", ` (i.e. with commas that need to be removed)

**Comment by za2201** [comment]
requested changes made :)

**Review by GCHQ-Developer-530 — DISMISSED** [review]

**Review by GCHQ-Developer-112 — APPROVED** [review]

**Review by GCHQ-Developer-112 — CHANGES_REQUESTED** [review]
Hi @za2201, please can you amend your commits so they're more reflective of the work you've completed

**Comment by za2201** [comment]
Hi, I've done this I think

**Comment by za2201** [comment]
> Hi @za2201, please can you amend your commits so they're more reflective of the work you've completed

Hi, I've done this I think

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by MI6-451 — DISMISSED** [review]

**Review by GCHQ-Developer-112 — DISMISSED** [review]

**Review by gd2910 — APPROVED** [review]

**Review by MI6-255 — DISMISSED** [review]

**Review by GCHQ-Developer-741 — APPROVED** [review]

**Review by GCHQ-Developer-847 — DISMISSED** [review]

**Review by ad9242 — DISMISSED** [review]

**Comment by gd2910** [comment]
Please see https://github.com/mi6/ic-design-system/pull/473

**Pull request closed without merging**